Signal/src/ViewControllers/ConversationView/Emoji Picker/EmojiPickerCollectionView.swift

```diff
@@ -318,6 +323,7 @@ extension EmojiPickerCollectionView: UICollectionViewDelegate {
 
         SDSDatabaseStorage.shared.asyncWrite { transaction in
             self.recordRecentEmoji(emoji, transaction: transaction)
+            emoji.baseEmoji.setPreferredSkinTones(emoji.skinTones, transaction: transaction)
```

Comment:
I tested locally without this line, and the skin tone preference was being persisted correctly. I think this line is unnecessary?